init-db

```diff
@@ -693,7 +693,7 @@ with dbwrapperType() as dbwrapper:
 		for db, dbData in data["databases"].items():
 			if type(dbData) == str:
 				# If it's a string, that's the user that owns it
-				dbData = { KEY_DB_PRIVILEGES : { mapUserName(username) : [ "*" ] } }
+				dbData = { KEY_DB_PRIVILEGES : { mapUserName(dbData) : [ "*" ] } }
```

Comment:
This code is actually correct. We're assigning the ONWERSHIP of the database, which means we need to map the privileges to a USER, not a database.

Reply:
I take it back. The suggested change is indeed correct :)